apps/datahub/src/app/home/news-page/key-figures/key-figures.component.html

```diff
@@ -4,7 +4,7 @@
       class="py-[37px] pl-[47px] rounded-lg border bg-white mb-5 card-shadow cursor-pointer"
       [figure]="recordsCount$ | async"
       [icon]="'folder_open'"
-      [title]="'catalog.figures.datasets'"
+      title="catalog.figures.datasets"
```

Comment:
why not use the `translate` pipe here, instead of `marker`? I now it's translated too in the gn-ui-figure component but amaybe that's the thing that doesn't make sense (translating inputs in presentation components mean the translation keys will not be discoverable)